apollo-rx2support/gradle.properties

```diff
@@ -0,0 +1,4 @@
+POM_ARTIFACT_ID=compiler
+POM_NAME=Apollo Android Compiler
+POM_DESCRIPTION=Implementation for the Gradle plugin
+POM_PACKAGING=jar
```

Comment:
it should be jar, and pure java module due to:
https://github.com/apollographql/apollo-android/pull/409